pkg/didcomm/protocol/didexchange/states.go

```diff
@@ -720,6 +821,18 @@ func (ctx *context) prepareConnectionSignature(connection *Connection,
 		return nil, fmt.Errorf("sign response message: %w", err)
 	}
 
+	if ctx.doACAPyInterop {
+		// Interop: aca-py expects raw base58 Ed25519 key instead of did:key
```

Comment:
please link to open issues in acapy.

Reply:
removed this path, as refactoring had made it dead code (and it was unnecessary)